Gcp-security-fundamentals-guide.md

```
@@ -0,0 +1,856 @@
+# Google Cloud セキュリティ基礎 完全ガイド
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)